package.json

```diff
@@ -129,8 +137,8 @@
 		"@fandom/context": "^0.6.1",
 		"@fandom/tracking-metrics": "0.0.28",
 		"@wikia/post-quecast": "2.0.1",
+		"clsx": "1.2.1",
 		"js-cookie": "3.0.1",
-		"rxjs": "7.5.4",
-		"styled-components": "^5.3.3"
+		"rxjs": "7.5.4"
```

Comment:
Is there a possibility of removing RxJS too ? its useful but length :/  and we can get some ~30kb/50kb improvement

Reply:
~20kB gzipped: https://bundlephobia.com/package/rxjs@7.8.1

Reply:
We'd need to replace the `communicationService` with some other solution as it requires RxJS here AFAIK. Do we have any general idea of replacing `communicationService` across all our apps and mainly AdEngine? If not yet, I'd put it into another ticket and keep the scope of this PR closed to removal of styled-components only 😊

Reply:
@nandy-andy We have this implementation to replace the rxjs in the communication service with native implementation in Silversurfer: https://github.com/Wikia/silver-surfer/pull/1470 which can be neat if replicated in your end.

Reply:
Well, @David-Iacovone it's worth trying if it's almost ready ;)

Reply:
It can definitely be part of another PR, as having this 18% improvement is good enough for now, David is already aware and working on it